src/parser/jobs/war/modules/InnerRelease.tsx

```diff
@@ -182,6 +198,32 @@ export default class InnerRelease extends Module {
 				severity: SEVERITY.MEDIUM,
 			}))
 		}
+
+		if (innerChaosUsed > 0) {
```

Comment:
Might be worth combining these two, as the abilities essentially share their concepts.

Can do later though.